File: C++/Matrizes/Soma_matrizes.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int M, N, i, j;

    cout << "Quantas linhas vai ter cada matriz? ";
    cin >> M;
    cout << "Quantas colunas vai ter cada matriz? ";
    cin >> N;

    int A[M][N], B[M][N], C[M][N];

    cout << "Digite os valores da matriz A: " << endl;

    for (i = 0; i < M; i++)
    {
        for (j = 0; j < N; j++)
        {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> A[i][j];
        }
    }

    cout << "Digite os valores da matriz B: " << endl;

    for (i = 0; i < M; i++)
    {
        for (j = 0; j < N; j++)
        {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> B[i][j];
        }
    }

    for (i = 0; i < M; i++)
    {
        for (j = 0; j < N; j++)
        {
            C[i][j] = A[i][j] + B[i][j];
        }
    }

    cout << "MATRIZ SOMA: " << endl;

    for (i = 0; i < M; i++)
    {
        for (j = 0; j < N; j++)
        {
            cout << C[i][j] << "  ";
        }

        cout << endl;
    }

    return 0;
}

File: Python/Matrizes/Soma_matrizes.py
M = int(input("Qual a quantidade de linhas da matriz? "))
N = int(input("Qual a quantidade de colunas da matriz? "))

A = [[0 for x in range(N)] for x in range(M)]
B = [[0 for x in range(N)] for x in range(M)]
C = [[0 for x in range(N)] for x in range(M)]

print("Digite os valores da matriz A:")

for i in range(0, M):
    for j in range(0, N):
        A[i][j] = int(input(f"Elemento [{i},{j}]: "))


print("Digite os valores da matriz B:")

for i in range(0, M):
    for j in range(0, N):
        B[i][j] = int(input(f"Elemento [{i},{j}]: "))


for i in range(0, M):
    for j in range(0, N):
        C[i][j] =  A[i][j] + B[i][j]


print("MATRIZ SOMA:")

for i in range(0, M):
    for j in range(0, N):
        print(C[i][j], end= "  ")

    print()


